app/assets/javascripts/advocates/claims/table_row_click.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+var adp = adp || {};
+
+adp.tableRowClick = {
+  $selector: null,
+
+  init : function(table_id) {
+    if($('#' + table_id).length) {
+      adp.tableRowClick.$selector = '#' + table_id;
```

Comment:
Why does it need an id? Class would be better because what if you had two tables on the page  you would not be able to give them the same ID as id need to be unique.